public/pages/Findings/containers/Findings/Findings.tsx

```diff
@@ -436,7 +439,8 @@ class Findings extends Component<FindingsProps, FindingsState> {
         const ruleLevel =
           finding.detectionType === 'Threat intelligence'
             ? 'high'
-            : (findingsState as DetectionRulesFindingsState).rules[finding.queries[0].id].level;
+            : (findingsState as DetectionRulesFindingsState).rules[finding.queries[0].id]?.level ||
```

Comment:
@jowg-amazon Just to clarify, do we need any null checks around `finding.queries`, or `finding.queries[0]`? I can't think of any situations when we would have a finding without at least 1 query, but wanted to throw that out there.

Reply:
This should be fine, a finding should always be associated with at least 1 query